### PR TITLE
SW-5652 Updated module details card to support application

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2855?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2855** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2855?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2853** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2853?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/Home/ParticipantHomeView/CurrentModule.tsx [hunk 1]
```diff
@@ -1,26 +1,66 @@
...
+      currentModule && currentParticipantProject
+        ? {
+            ...currentModule,
+            title: activeLocale ? strings.formatString(strings.TITLE_OVERVIEW, currentModule.title).toString() : '',
+          }
+        : null,
+    [activeLocale, currentModule]
+  );
+
+  return currentParticipantProject && moduleDetails ? (
```
Comment (nickgraz) [inline src/scenes/Home/ParticipantHomeView/CurrentModule.tsx:55]: If you prefer:
```suggestion
  if (!(currentParticipantProject && moduleDetails)) {
    return null
  }
```

**Review by nickgraz — APPROVED** [review]
Looks good to me!

**Comment by tommylau523** [comment]
### Merge activity

* **Jul 15, 11:10 PM EDT**: @tommylau523 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2855).

**Pull request merged**